app/kubernetes/components/yaml-inspector/yamlInspectorController.js

```diff
@@ -39,6 +39,10 @@ class KubernetesYamlInspectorController {
     this.expanded = !this.expanded;
   }
 
+  onChange() {
```

Comment:
is this change required for the tooltip ticket?

Reply:
removed